### DISCUSSION TITLE
Updating content for a quarto book using R

### DISCUSSION THREAD

**Discussion by ark561** [quarto-dev/quarto-cli#5305] [Q&A]
Hi everyone,

I apologize for the somewhat basic questions, but I am unsure of how to update content on my GitHub after I have made changes to a quarto book that I am constructing in R.

If I make a change to a file, such as summary_stats.qmd, how would I update the file for the GitHub site? Do I have to recompile the docs folder, and then delete/update that via a git command?

Thanks so much in advance!!

**Comment by mcanouil** [comment]
Hard to really answer without knowing what's your setup.
I suggest you read <https://quarto.org/docs/publishing/github-pages.html>.

**Reply by ark561** [reply]
I'm using R Studio, and using the Terminal window through R Studio to work with Github. I've been using the link you posted, and got an initial site set up by creating and uploading a docs folder. 

Now, after updating some of the files on my local machine, I want to copy them over/replace the files that are currently on Github. I'm unsure of how to actually update the files using git commands.

**Reply by mcanouil** [reply]
Take a look at <https://happygitwithr.com/rstudio-git-github.html> or other online resources to learn to use Git and Github, such as GitHub documentation for example.
